File: C++/Day 5/sortArr.cpp
#include<iostream>
using namespace std;
void sortArr(int arr[],int n){
	for(int i=0;i<=n-2;i++){
		for(int j=i+1;j<=n-1;j++){
			if(arr[i]>arr[j]){
				int temp= arr[i];
				arr[i]=arr[j];
				arr[j]=temp;
			}
		}
	}
}

void displaySort(int arr[],int n){
	for(int k=0;k<n;k++){
		cout<<arr[k]<<" ";
	}
}

int main(){
	int n;
	cout<<"Enter number="<<endl;
	cin>>n;
	int *arr = new int[n];
	for(int i=0;i<n;i++){
		cin>>arr[i];
	}
	sortArr(arr,n);
	displaySort(arr,n);
	return 0;
}

File: Java/Day 2/src/SortArr.java
import java.util.Scanner;
public class SortArr{
	public static void Arrsort(int arr[]){
		for(int i=0;i<9;i++){
			for(int j=i+1;j<10;j++){
					if(arr[i]>arr[j]){
						int temp=arr[i];
						arr[i]=arr[j];
						arr[j]=temp;
					}
			}
		}
			
	}
	
	public static void main(String[] args){
			Scanner sc = new Scanner(System.in);
			System.out.println("Enter 10 elements: ");
			int arr[]=new int[10];
			for(int i=0;i<10;i++){
				arr[i] = sc.nextInt();
			}
			Arrsort(arr);
			System.out.println("Sorted Array:");
			for(int i=0;i<10;i++){
				System.out.print(arr[i]+ " ");
			}
	}
}